.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+before_install:
+  - gem install fpm
+  - pip install awscli
+
+script: make deb
+
+before_deploy:
+  - aws s3 sync s3://mirror.cs50.net/ide50/ .travis/ide50/
```

Comment:
Copy of the contents on the bucket, fetched before deploying. Since we're manually doing that, `dpkg-scanpackages`, which generates `Packages` file, needs all packages in the directory, to parse their metadata, and include them in `Packages`. Otherwise, it would only include the package to be deployed, which would break the repo.